CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```

Comment:
We would have to discuss with the team about the code of conduct. Can't accept something just because it's used by other projects.

Reply:
Okay, that's fair enough. Was just an idea since I noticed it today. Happy to close this.